patches/minecraft/net/minecraft/village/Village.java.patch

```diff
@@ -0,0 +1,72 @@
+--- ../src-base/minecraft/net/minecraft/village/Village.java
++++ ../src-work/minecraft/net/minecraft/village/Village.java
+@@ -26,7 +26,7 @@
+ import net.minecraft.util.math.Vec3d;
+ import net.minecraft.world.World;
+ 
+-public class Village
++public class Village implements net.minecraftforge.common.capabilities.ICapabilitySerializable<NBTTagCompound>
+ {
+     private World field_75586_a;
+     private final List<VillageDoorInfo> field_75584_b = Lists.<VillageDoorInfo>newArrayList();
+@@ -43,11 +43,13 @@
+ 
+     public Village()
+     {
++        this(null);
```

Comment:
This seems slightly questionable? May be better to just set the capabilities here too. I'm open to other opinions though.